cmd/db/dynamodb.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"fmt"
+	"sync"
+	
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/pokt-network/pocket-core/const"
+)
+
+// NOTE: You need to setup your AWS configuration
+// first https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html
+
+const (
+	REGION   = "us-west-2"
+	ENDPOINT = "http://localhost:8000"
+)
+
+var (
+	c    *aws.Config
+	once sync.Once
+)
+
+func config() *aws.Config {
+	once.Do(func() {
+		c = &aws.Config{
+			Region:   aws.String(REGION),
+			Endpoint: aws.String(ENDPOINT),
+		}
+	})
+	return c
+}
+
+func DB() *dynamodb.DynamoDB {
+	return dynamodb.New(session.Must(session.NewSession(config())))
+}
+
+func main() {
+	var i int
+	fmt.Print("(1) Create DP Table:\n(2) Delete DP Table: ")
+	_, err := fmt.Scanf("%d", &i)
+	if err != nil {
+		fmt.Println(err)
+	}
+	switch i {
+	case 1:
+		CreateTable()
+	case 2:
+		DeleteTable()
+	}
```

Comment:
This behaves better as part of a while loop, starting at about line 42, and with a default case that would advise the user to choose either 1 or 2 depending on the situation and restart.

Reply:
Agreed ^ will fix this